cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -89,3 +89,8 @@ As a workaround, create the template from the command line.
 importing VMware virtual machines, and creating virtual machines by using the
 wizard.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1724654[*BZ#1724654*])
+
+* When you install {CNVProductName}, kubemacpool automatically starts. 
+If you define a secondary vNIC without specifying the MAC address, the MAC pool allocates
+a unique address to the vNIC. However, if you do not define a secondary vNIC and
+specify the MAC address, the secondary vNIC may collide with another vNIC in the cluster.
```

Comment:
s/may/might - or s/may/can

>! may v
Use only to indicate permission. Otherwise, use “can” or “might.”